CHANGELOG/CHANGELOG-2.3.md

```diff
@@ -0,0 +1,381 @@
+# Release notes for v2.3.0
```

Comment:
This is going to be 3.0.0 because we're bumping the min k8s version to 1.20. The readme should also be updated.